README.md

```diff
@@ -69,6 +69,33 @@ config :airbrake,
   ignore: :all # to disable reporting
 ```
 
+## Shared options for reporting data to Airbrake
+
+To include with every report to Airbrake a set of optional data, include the `:options` key in the config. This can either
+be a keyword list of options or a function that returns a keyword list of options. Keyword list keys that can be used are
+`:context`, `:params`, `:session`, and `:env`.
+
+### Options function in config
+
+A function for creating options for reporting should be declared in the config as a tuple of 
+`{ModuleName, :function_name, 1}`. This function should take as an argument a keyword list, possibly empty and should
+return a keyword list. The function arity is always 1.
+
+```elixir
+config :airbrake,
+  options: {Web, :airbrake_options, 1}
+```
+
+### Options keyword list in config
+
+When options are provided as a keyword list in the configuration and a specific call to `Airbrake.report/2` includes 
+options in its parameters, the options will be merged, with the parameters taking precedence.
+
+```elixir
+config :airbrake,
+  options: [env: %{"SOME_ENVIRONMENT_VARIABLE" => "environment variable"}]
```

Comment:
Can we please rephrase these to not mention environment variables specifically? I fear this is misleading due to the additional amount of work necessary to read a true environment variable at boot-time in our existing Elixir practices - only a few of the services are set up correctly to do so, and it's not implemented uniformly across that list, either. You can't just use `"$MY_ENV"` as the literal value and have it work out-of-the-box in most our projects.

I'd suggest using less-abstract/meaningful, but still static, keys and values - perhaps something like:
```elixir
  options: [env: %{"NAMESPACE" => "my-team", "ENVIRONMENT_NAME" => "uat"}]
```

Reply:
Spoke with Cliff verbally a few minutes ago and pending some follow-up reference links to confirm our understanding, I'm fine with merging this as-is.

Reply:
The [API for Airbrake](https://airbrake.io/docs/api/#create-notice-v3) describes the object as 

> An object containing the current environment variables. Where the key is the variable name, e.g. { "PORT": "443", "CODE_NAME": "gorilla" }.